### PR TITLE
1.10 TRILAB-266: Remove clang warnings.

### PR DESCRIPTION
**Pull request by hyoklee**
The same patch was applied to develop & 1.12 branches and merged.

### CODE REVIEW

File: config/cmake/cacheinit.cmake [hunk 1]
```diff
@@ -67,7 +67,7 @@ set (PLUGIN_PACKAGE_NAME "pl" CACHE STRING "Name of PLUGIN package" FORCE)
 # bitshuffle
 ###########
 
-set (BSHUF_GIT_URL "https://git@bitbucket.hdfgroup.org/scm/test/bitshuffle.git" CACHE STRING "Use BSHUF from HDF repository" FORCE)
+set (BSHUF_GIT_URL "https://github.com/hyoklee/bitshuffle.git" CACHE STRING "Use BSHUF from HDF repository" FORCE)
```
Comment (byrnHDF) [inline config/cmake/cacheinit.cmake]: I don't remember these changes.

File: test/cork.c [hunk 2]
```diff
@@ -89,10 +89,11 @@ static unsigned
 verify_old_dset_cork(void)
 {
     /* Variable Declarations */
-    hid_t      fid = -1;                             /* File ID */
-    hid_t      did = -1, did2 = -1, did3 = -1;       /* Dataset IDs */
-    hid_t      dcpl = -1, dcpl2 = -1, dcpl3 = -1;    /* Dataset creation property lists */
-    hid_t      sid = -1, sid2 = -1, sid3 = -1;       /* Dataspace IDs */
+    hid_t fid = H5I_INVALID_HID;                                                 /* File ID */
+    hid_t did = H5I_INVALID_HID, did2 = H5I_INVALID_HID, did3 = H5I_INVALID_HID; /* Dataset IDs */
```
Comment (byrnHDF) [inline test/cork.c:93]: I still think these should be individual lines, but won't complain if they stay as is.

File: test/cork.c [hunk 3]
```diff
@@ -295,19 +296,19 @@ static unsigned
...
-    int        i            = 0;     /* Local index variable */
-    hsize_t    dim[1]       = {100}; /* Dataset dimension size */
-    hsize_t    chunk_dim[1] = {7};   /* Dataset chunk dimension size */
-    H5O_info_t oinfo, oinfo2;        /* Object metadata information */
-    char       attrname[500];        /* Name of attribute */
-    unsigned   flags;                /* File access flags */
+    hid_t      fid  = H5I_INVALID_HID;                        /* File ID */
+    hid_t      fapl = H5I_INVALID_HID;                        /* File access property list */
+    hid_t      aid  = H5I_INVALID_HID;                        /* Attribute ID */
+    hid_t      sid = H5I_INVALID_HID, sid2 = H5I_INVALID_HID; /* Dataspace IDs */
```
Comment (byrnHDF) [inline test/cork.c:302]: Same here

**Review by byrnHDF — CHANGES_REQUESTED** [review]
I only have an issue with the cacheinit file change. Don't we have a HDF repo?

**Review by byrnHDF — APPROVED** [review]

File: test/cork.c (hunk 2 above)
Comment (lrknox) [reply test/cork.c:93]: Agreed, but probably the change should start in develop and then be merged down to the other branches?

**Review by lrknox — APPROVED** [review]

**Pull request merged**